### ISSUE TITLE
Interested in TypeScript?

### ISSUE THREAD

**Issue by dwilhelmi** [ember-animation/ember-animated#89]
Hi! I learned about this addon at EmberConf, and I'm really wanting to start using it. I’m also participating in [this quest issue to add TypeScript support throughout the Ember.js ecosystem](https://github.com/typed-ember/ember-typings/issues/14). This will benefit to *everyone* who uses your addon, not just TypeScript users! (See [here]( improving-developer-experience-for-all-emberjs-users) for an explanation.)

Are you interested in either converting the addon to use TypeScript itself, or in adding type definitions? I’d love to help out, if so! And if not, that’s just fine. Thanks!

**Comment by ef4** [comment]
Yes, and I already have a branch where I started converting. See
https://github.com/ember-animation/ember-animated/tree/ts

I haven’t picked that work up lately so feel free to keep building off that
branch. I was doing a file by file conversion, keeping CI green and the
whole typecheck correct.
